license-maven-plugin-git/src/main/java/com/mycila/maven/plugin/license/git/GitLookup.java

```diff
@@ -86,35 +86,15 @@ public enum DateSource {
    * @return      the git lookup
    */
   public static GitLookup create(File file, Map<String, String> props) {
-    final GitLookup.DateSource dateSource = Optional.ofNullable(props.get(COPYRIGHT_LAST_YEAR_SOURCE_KEY))
-        .map(String::trim)
-        .map(String::toUpperCase)
-        .map(GitLookup.DateSource::valueOf)
-        .orElse(GitLookup.DateSource.AUTHOR);
+    final GitLookup.DateSource dateSource = Optional.ofNullable(props.get(COPYRIGHT_LAST_YEAR_SOURCE_KEY)).map(String::trim).map(String::toUpperCase).map(GitLookup.DateSource::valueOf).orElse(GitLookup.DateSource.AUTHOR);
```

Comment:
can you please revert to the original format of the file and keep the existing code style of a source code ? this is to keep the diffs focused on the PR goal - no code reformat in the PRs. ;-) Thanks!

Reply:
Done